example/compose/nginx/portal/nginx.conf.ejs

```diff
@@ -35,10 +35,15 @@ http {
 
         # Serve the backend API for the portal.
         location /api {
-            proxy_set_header  Host $http_host;
```

Comment:
This is breaking the links in identity validation emails. it should be replaced by X-Forwarded-Host instead.